### PR TITLE
[MODULAR] Adds a spawntext to the security medic, clarifying their medical knowledge.

### PR DESCRIPTION
**Pull request by RatFromTheJungle**
## About The Pull Request

fix4host

## Why It's Good For The Game
Fixes security medics being a better medical department.
## Changelog
no

### CODE REVIEW

**Comment by thestubborn** [comment]
aww
there goes me being an inhouse surgery man
(I mean paramedics shouldn't even know how to do surgery)

File: modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm [hunk 1]
```diff
@@ -25,6 +25,10 @@
 
 	family_heirlooms = list(/obj/item/clothing/neck/stethoscope, /obj/item/roller, /obj/item/book/manual/wiki/security_space_law)
 
+/datum/job/security_medic/after_spawn(mob/living/carbon/human/H, mob/M)
```
Comment (tf-4) [inline modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm:28]: what is this H M bullshit bro u should know better than that

Comment (RatFromTheJungle) [reply modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm:28]: i copied how gandalf did it bro....

Comment (tf-4) [reply modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm:28]: newbie error

**Pull request merged**